### PR TITLE
Fixed circuit fit method passing bug and added testing for L and T el…

### PR DESCRIPTION
**Pull request by BGerwe**
Addresses the minor bug in #61

### CODE REVIEW

**Comment by mdmurbach** [comment]
Awesome!

File: impedance/circuits.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def __eq__(self, other):
         else:
             raise TypeError('Comparing object is not of the same type.')
 
-    def fit(self, frequencies, impedance, method='lm', bounds=None):
+    def fit(self, frequencies, impedance, method=None, bounds=None):
```
Comment (mdmurbach) [inline impedance/circuits.py]: I think we should leave the default as lm (that way we can still just call .fit(f, Z) without having to pass a method every time)

Comment (BGerwe) [reply impedance/circuits.py]: Leaving lm as default actually breaks things, I think. If None is passed to method scipy will default to trf/lm for a function call with/without bounds.

Comment (mdmurbach) [reply impedance/circuits.py]: Ah, gotcha :+1:

**Comment by mdmurbach** [comment]
Thanks @BGerwe!!!

**Pull request merged**